src/Take.Blip.Builder/Utils/ContactMomoryCache.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using Lime.Messaging.Resources;
+using Microsoft.Extensions.Caching.Memory;
+
+namespace Take.Blip.Builder.Utils
+{
+    public class ContactMomoryCache : ICache<Contact>
```

Comment:
Fix typo (`Momory`)

Reply:
Great. I will add cache using an decorator.

Reply:
I created another pull request since I needed to change code from other project inside this repository. 

https://github.com/takenet/blip-sdk-csharp/pull/23